src/services/organization-extension.js

```diff
@@ -119,7 +147,35 @@ module.exports = class orgExtensionsHelper {
 				})
 			}
 
-			const { review_stages, review_type } = bodyData
+			const { review_stages, review_type, data_managers } = bodyData
+
+			// check if body have data_managers
+			if (data_managers) {
```

Comment:
@adithyadinesh0412 There should be a single query of create in this function. cc - @priyanka-TL

Reply:
updated